### PR TITLE
Add Sicredi as adopter

### PR DESCRIPTION
**Pull request by igorestevanjasinski**
Adding Sicredi as an opentelemetry Adopter

### CODE REVIEW

File: data/ecosystem/adopters.yaml [hunk 1]
```diff
@@ -73,6 +73,11 @@
   components: [Collector, Go, Ruby]
   reference: ''
   contact: ''
+- name: Sicredi
+  url: https://www.sicredi.com.br/
+  components: [Collector, Go, Java, Python]
+  reference: ''
+  contact: ''
```
Comment (svrnm) [inline data/ecosystem/adopters.yaml:80]:
```suggestion
  reference: ''
  contact: ''
```

Do you have any blog post, conference talk (recording, slides, etc.) we can provide as reference?

for the contact, please put here either your github handle (preferred) or an email address we can reach out to if we have questions, e.g. if any link throws 404s or if we plan to apply significant changes.

Comment (igorestevanjasinski) [reply data/ecosystem/adopters.yaml]: Hey Severin, we have only internal material, but I'm working to publish a blog post about it. Can I update the reference soon?

File: data/ecosystem/adopters.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # cSpell:ignore Farfetch Globale Datenrettungsdienste Uplight Wandera Zocdoc Skyscanner
 - name: AppDirect
-  url: https://www.appdirect.com/
+  url: https://www.appdirect.com/agoago
```
[the remark this answers is not in the document]
Comment (igorestevanjasinski) [reply data/ecosystem/adopters.yaml]: No, sorry. I removed, was a typo

[the remark this answers is not in the document]
Comment (svrnm) [reply data/ecosystem/adopters.yaml]: 👍

**Comment by svrnm** [comment]
@igorestevanjasinski apologies for the delay here, the blocker is resolved and I will merge this PR now

**Review by svrnm — APPROVED** [review]

**Comment by igorestevanjasinski** [comment]
> @igorestevanjasinski apologies for the delay here, the blocker is resolved and I will merge this PR now

no problem, thank you @svrnm

**Pull request merged**